db/migrate/20210715154417_change_cpf_telefone_to_be_string_in_cidadaos.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeCpfTelefoneToBeStringInCidadaos < ActiveRecord::Migration[6.0]
+  def change
+    change_column :cidadaos, :telefone, :string
```

Comment:
Adiciona índices únicos aos campos telefone e cpf. Assim, você consegue validar de fato via banco, e, add a camada de validação no model (validades uniqueness ..)

Essa leitura dá uma melhor clareza: https://thoughtbot.com/blog/the-perils-of-uniqueness-validations

Reply:
Sim, já li a documentação, já faço isso sim.